src/components/Card/README.md

```diff
@@ -266,7 +266,29 @@ Use footer actions for a card’s most important actions, or actions merchants s
 ```jsx
 <Card
   title="Shipment 1234"
-  secondaryFooterAction={{content: 'Cancel shipment'}}
```

Comment:
as the old API is deprecated, I figured I'd update the existing singular example

Not sure why the lines below are showing up as new though